docs/code_contribution_guidelines.md

```diff
@@ -215,7 +215,7 @@ being provided here.
 Here’s a model Git commit message:
 
 ```
-Short (50 chars or less) summary of changes
+prefix: Short summary of changes (50 chars or less)

```

Comment:
The model is no longer 50 chars!

If you change `(50 chars or less)` to `(max 50 chars)` it will fit.